examples/web/index.html

```diff
@@ -490,6 +490,20 @@ <h2>CheckboxField</h2>
           </div>
         </div>
       </section>
+
+      <section>
+        <div class="Container">
+          <h2>Alert</h2>
```

Comment:
Please add `typography-heading-medium-text` class to this and previous headline (Checkbox) so they match other headlines in the demo, both here and Jobs. Thank you!

Reply:
Fixed.